RealmSwift/Tests/ListTests.swift

```diff
@@ -148,7 +148,7 @@ class ListTests: TestCase {
         XCTAssertEqual(str2, array[0])
         XCTAssertEqual(str1, array[1])
 
-        assertThrows(self.array[200])
+        assertThrows(self.array[200], named: nil)
```

Comment:
Are we using `named: nil` to indicate that `assertThrows` should not check the exception name? It would be clearer to use the version without the name argument for that purpose.